api/handlers_movers.go

```diff
@@ -285,8 +282,28 @@ func (s *server) RunShowHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	respAll, errAll := orch.datamoverRunList(r.Context(), group, name, true)
```

Comment:
The only thing you should be calling in this handler is the `orch.datamoverRunDescribe` function and all the logic should be in there, so call it like this:
```
resp, err := orch.datamoverRunDescribe(r.Context(), group, name, id)
```

Reply:
This method we are using twice  
/{account}/movers/{group}/{name}/runs/{id} ---As the ID will come Truncated , we need full ARN to get information.